src/loot_mover.py

```diff
@@ -98,7 +98,9 @@ def _move_items(
         if (
             (move_item_type == MoveItemsType.favorites and item.is_fav)
             or (move_item_type == MoveItemsType.junk and item.is_junk)
-            or (move_item_type == MoveItemsType.non_favorites and not item.is_fav)
+            or (move_item_type == MoveItemsType.non_favorites and (
```

Comment:
Why do we need to change the loot mover?

Reply:
Right now `non_favorite` items also move junk. We already have a separate entry for `junk` . I decided to fix this when `mark_as_favorte` is set to `false`, because that'll be the only time where items will have normal state that is not favorite and not junk. I hope that makes sense.

Reply:
I think I get it, but would it be more clear to just add an "unmarked" option to the loot mover instead? I don't think I love the idea of it working different ways depending on what's set for a completely unrelated property.

Reply:
+1 for just adding all possibilities as is:
- junk
- favorites
- plain
- all

Reply:
thanks both, I will add take a stab at this